frontend/src/Components/Navbar.tsx

```diff
@@ -72,95 +70,290 @@ export default function Navbar({
                         {user && isAdministrator(user) ? (
                             <>
                                 {/* admin view */}
-                                {hasFeature(
+                                {user.feature_access.length === 1 &&
+                                hasFeature(
                                     user,
                                     FeatureAccess.OpenContentAccess
-                                ) && user.feature_access.length === 1 ? (
+                                ) ? (
                                     <li className="mt-16">
                                         <Link to="/knowledge-center-dashboard">
                                             <ULIComponent icon={HomeIcon} />
-                                            Dashboard
+                                            Knowledge Center
                                         </Link>
                                     </li>
                                 ) : (
                                     <li className="mt-16">
                                         <Link to="/admin-dashboard">
                                             <ULIComponent icon={HomeIcon} />
-                                            Dashboard
+                                            {user.feature_access.length === 1 &&
+                                            hasFeature(
+                                                user,
+                                                FeatureAccess.ProviderAccess
+                                            )
+                                                ? 'Student Activity'
+                                                : 'Dashboard'}
                                         </Link>
                                     </li>
                                 )}
-                                {hasFeature(
-                                    user,
-                                    FeatureAccess.OpenContentAccess
-                                ) && (
+
+                                {/* Check for 0 features selected and render the resources link if true */}
+                                {user.feature_access.length === 0 && (
```

Comment:
Is this rendering the link to the knowledge-center-management if there are no features enabled?

Reply:
@PThorpe92  I don't understand what you are asking?

Reply:
@corypride Resource Management was moved over to the Knowledge Center and is now called Helpful Links (#526)